crates/payout_curve/examples/payout_curve_csv.rs

```diff
@@ -25,9 +26,17 @@ use trade::Direction;
 fn main() -> Result<()> {
     let initial_price = dec!(30_000);
     let quantity = 30_000.0;
-    let leverage_short = 3.0;
+    let leverage_short = 2.0;
     let leverage_long = 2.0;
-    let fee = 0;
+    // fee is 0.3% * quantity/initial_price = 0.003 BTC = 300_000 sats. We compute it here so that
```

Comment:
🔧 the comment does not match the implementation.

Is it `quantity / initial_price` or `initial_price / quantity`

Reply:
Good catch. It just works because `quantity` = `initial_price` 🙈. It is `quantity / initial_price`